packages/blocks/src/api/parser/apply-built-in-validation-fixes.js

```diff
@@ -32,8 +37,14 @@ export function applyBuiltInValidationFixes( block, blockType ) {
 		originalContent
 	);
 
-	return {
+	const updatedBlock = {
 		...block,
 		attributes: updatedBlockAttributes,
 	};
+
+	return applyFilters(
+		'blocks.applyBuiltInValidationFixes',
```

Comment:
This hook is necessary to handle deprecation from stored content.

It is useful when handling deprecation from stored content.